lib/utilities/will-interrupt-process.js

```diff
@@ -143,9 +145,9 @@ function setupSignalsTrap() {
  * @method teardownSignalsTrap
  */
 function teardownSignalsTrap() {
-  _process.removeListener('SIGINT', exit);
-  _process.removeListener('SIGTERM', exit);
-  _process.removeListener('message', onMessage);
+  _process.off('SIGINT', exit);
```

Comment:
Using `off` is more in line with using `on` above. `removeListener` is an alias for `off`.